l10n_br_account_payment_brcobranca/parser/cnab_file_parser.py

```diff
@@ -360,30 +360,26 @@ def process_return_file(self, data):
                 # 'tipo_moeda': evento.credito_moeda_tipo,
             }
 
-            # Caso de Pagamento deve criar os Lançamentos de Diário
-            if cod_ocorrencia in cnab_liq_move_code:
-                row_list, log_event_payment = self._get_accounting_entries(
-                    linha_cnab, account_move_line, payment_lines
-                )
-                result_row_list.append(row_list)
-                cnab_return_log_event.update(log_event_payment)
-            else:
-                # Nos codigos de retorno cadastrados no Data do modulo
+            row_list, log_event_payment = self._get_accounting_entries(
+                linha_cnab, account_move_line, payment_lines, cod_ocorrencia
+            )
+            result_row_list.append(row_list)
+            cnab_return_log_event.update(log_event_payment)
+            if cod_ocorrencia not in cnab_liq_move_code:
+                # Nos códigos de retorno cadastrados no Data do módulo
                 # l10n_br_account_payment_order o 02 se refere a
                 # Entrada Confirmada e 03 Entrada Rejeitada.
                 # TODO: Estou considerando que seja um padrão, existem
-                #  exceções ?
-                #  Caso exista será preciso criar o campo no payment.mode
-                #  para informa-lo como nos outros casos.
+                # exceções ?
+                # Caso exista será preciso criar o campo no payment.mode
+                # para informá-lo como nos outros casos.
                 if cod_ocorrencia == "02":
                     account_move_line.cnab_state = "accepted"
                 elif cod_ocorrencia == "03":
                     # TODO - algo a mais a ser feito ?
                     account_move_line.cnab_state = "not_accepted"
-
-            # Inclui o LOG do Evento CNAB
-            self.cnab_return_events.append(cnab_return_log_event)
-
+                # Inclui o LOG do Evento CNAB
+                self.cnab_return_events.append(cnab_return_log_event)
```

Comment:
@CristianoMafraJunior to preocupado com essa alteração aqui, de colocar esse `append` dentro do bloco do `if`, será que tá certo mesmo isso ? não deveria estar fora do bloco do `if` igual era antes? ou será que tem algo que não to pegando bem, se puder verificar.

Reply:
acho que quando estava mexendo ali acabei deixando dentro do If, mas arrumei